docs/layers/tile-3d-layer.md

```diff
@@ -122,6 +122,12 @@ const layer = new Tile3DLayer({
 })
 ```
 
+##### `pickable` (Boolean, Optional)
+
+- Default: false
+
+When enable [`picking`](/docs/developer-guide/custom-layers/picking.md), the picking info will contain the selected [`tile`](https://github.com/uber-web/loaders.gl/blob/master/modules/3d-tiles/docs/api-reference/tileset-3d.md) accessed by`info.object`.
```

Comment:
- You are linking to the `Tileset3D` class which is not what `tile` is.
- Link to the website please, master can contain unpublished changes.
- Wording: `info.object` will be a `Tile3DHeader` (link) object.

Reply:
only the incomplete category spec `https://loaders.gl/docs/specifications/category-3d-tiles#tileheader-fields` is available at the moment.